options/options_helper.cc

```diff
@@ -969,6 +969,20 @@ Status OptionTypeInfo::Parse(const ConfigOptions& config_options,
       ConfigOptions copy = config_options;
       copy.invoke_prepare_options = false;
       return parse_func_(copy, opt_name, opt_value, opt_addr);
+    } else if (opt_name == "read_amp_bytes_per_bit") {
+      // A temporary hack to work around a bug in 6.10, 6.11, 6.12, 6.13 and
+      // 6.14. The bug will write out 8 bytes to OPTIONS file from the starting
```

Comment:
Can we tone down the comment to just say "A workaround to fix a bug in earlier releases where 8 bytes...